BibBuddy/.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
-    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default copy1" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Checker" />
   </state>
-</component>
+</component>
```

Comment:
Please add a new line at the end of the file.

Reply:
I don't think I should alter a file created by Android Studio itself. Also, when attempting to commit this, GitHub itself warns me of doing this - therefore not included in my commit.